meta/autoid/autoid.go

```diff
@@ -80,7 +80,11 @@ type Allocator interface {
 	// Alloc allocs N consecutive autoID for table with tableID, returning (min, max] of the allocated autoID batch.
 	// It gets a batch of autoIDs at a time. So it does not need to access storage for each call.
 	// The consecutive feature is used to insert multiple rows in a statement.
-	Alloc(tableID int64, n uint64) (int64, int64, error)
+
+	// increment & offset is used to valid the start position (the allocator's base is not always the last allocated id).
+	// The returned range is still (min, max], you need derive the ids like firstID, firstID + increment * 2... in caller.
```

Comment:
```suggestion
	// The returned range is still (min, max], you need derive the ids like firstID, firstID + increment * 2... in caller.
```
Please update the comment, "the returned range is _still (min,max]_" is not proper now.

Reply:
the allocate logic is still the same, return (min, max],
the caller's responsibility is to derive the autoid series from this range
 refer: `AllocBatchAutoIncrementValue`